functions/openhab.bash

```diff
@@ -14,14 +30,12 @@ delayed_rules() {
 
   targetDir="/etc/systemd/system/openhab2.service.d"
 
+  create_sys_dependencies
   if [[ $1 == "yes" ]]; then
     echo -n "$(timestamp) [openHABian] Adding delay on loading openHAB rules... "
-    if ! cond_redirect mkdir -p $targetDir; then echo "FAILED (prepare  directory)"; return 1; fi
-    if ! cond_redirect rm -f "${targetDir}"/override.conf; then echo "FAILED (clean directory)"; return 1; fi
-    if cond_redirect cp "${BASEDIR:-/opt/openhabian}"/includes/systemd-override.conf "${targetDir}"/override.conf; then echo "OK"; else echo "FAILED (copy configuration)"; return 1; fi
+    if cond_redirect cat "${BASEDIR:-/opt/openhabian}"/includes/delayed-rules.conf >>"${targetDir}"/override.conf; then echo "OK"; else echo "FAILED (copy configuration)"; return 1; fi
   elif [[ $1 == "no" ]]; then
```

Comment:
Please use sed or some other program to make this actually work and disable delayed rules.

Reply:
I thought about but sed is really hard to get right because there's multi-line dependencies ... I possibly would not get that right right away. So this is the first attempt at a multi-line building block approach. Works for delayed rules but no additional blocks ATM. Let's keep that and extend to multiple blocks (to use a generator function with a table of functions to include) in a future patch.

Reply:
Perhaps delete the current and restore the original `override.conf` without adding the delayed rules.

Reply:
that's what the current code does, doesn't it? It always (oninstall,  add or remove rules) creates it as the original `override.conf`, then if add it appends the rules part.

Reply:
Oh, I see what you are saying now, I did not notice that before.